guide/05-working-with-the-spatially-enabled-dataframe/part3_data_io_writing_data.ipynb

```diff
@@ -0,0 +1,1816 @@
+{
```

Comment:
<p>The data used in this guide is provided as an <a href="https://www.arcgis.com/home/item.html?id=518fcbd5ef7440a0877726b0ac95b421" target="_blank">item</a>. We will start by importing some libaries and downloading and extracting the data needed for the analysis in this guide. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>